.github/workflows/poetry_lockfile.yaml

```diff
@@ -0,0 +1,24 @@
+on:
+  push:
+    branches: ["develop", "release-*"]
+    paths:
+      - poetry.lock
+  pull_request:
+    paths:
+      - poetry.lock
```

Comment:
Nice that we only trigger this if needed.

The downside is that it can't be fed into the `tests-done` job to tell github it's safe to merge this PR. But I think we can live with that.

Reply:
It's frustrating how the default behaviour isn't just to make sure all the tests pass, but ah well